doc/release/RELEASE-PROTOCOL.md

```diff
@@ -106,25 +106,24 @@ Hotfix branches are utilized to quickly patch production and sandbox releases. T
 
 ### Duration of the development of the new release version
 
-The development of the new release version takes four weeks before it is released.
+The development of the new release version takes two weeks before it is released.
 
 
 ### Introductory actions after starting the development cycle of the new release version
 
 Make sure that:
-1. The version planned for release in four weeks is created in Jira and is indicated in all Jira tickets related to the upcoming release.
+1. The version planned for release in two weeks is created in Jira and is indicated in all Jira tickets related to the upcoming release.
 2. The start and release dates are set on the page of the release in Jira.
 3. The draft with changes for the upcoming release is added to the RELEASE-NOTES.md file. Ensure the version of the release and its date are set for the change draft.
 4. Verify that the release schedule in the RELEASE-PROTOCOL.md file is correctly filled.
 
 
 ### Code Freeze
 
-1. Code freeze occurs every Monday in the fourth week of new release version development. To enhance communication within the team, all stakeholders must be notified about the code freeze two working days before the code freeze by the responsible person for the release.
-2.  On the day of the code freeze, the responsible person has to verify that the pre-prod environment is up and running. The responsible person needs to change the database name of the existing `opensupplyhub-enc-pp` database to `opensupplyhub-enc-pp-current` via Amazon RDS in the Databases section.
+1. Code freeze occurs every Monday in the second week of new release version development. To enhance communication within the team, all stakeholders must be notified about the code freeze two working days before the code freeze by the responsible person for the release.
```

Comment:
> Code freeze occurs every Monday in the second week 

I believe it should be "in the third week," not "in the second week," because it will occur after every two-week sprint.